cypress.json

```diff
@@ -2,5 +2,6 @@
   "screenshotsFolder": "tmp/cypress_screenshots",
   "trashAssetsBeforeRuns": false,
   "video": false,
-  "retries": 3
+  "retries": 3,
+  "chromeWebSecurity": false
```

Comment:
This is needed to be able to test within iframes - see https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/